webapp/graphite/tags/redis.py

```diff
@@ -151,29 +156,44 @@ def get_series(self, path, requestContext=None):
 
     return TaggedSeries(tags['name'], tags)
 
-  def list_tags(self, tagFilter=None, requestContext=None):
-    return sorted([
+  def list_tags(self, filter=None, limit=None, requestContext=None):
```

Comment:
here the filter is not anchored at the beginning. that might be fine if it's done before calling `list_tags()`, but it's kind of inconsistent with the `local_database` tagdb then

Reply:
`re.match` is implicitly anchored